conda/recipes/rmm/meta.yaml

```diff
@@ -33,7 +34,7 @@ requirements:
     - python
     - numba >=0.49
     - numpy
-    - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
+    - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
```

Comment:
In conda-forge we decided that the `min_pin` should be `x.x`. Please see comment ( https://github.com/conda-forge/nvcc-feedstock/pull/71#issuecomment-930228558 ) and subsequent discussion for context.

Reply:
Hmm. I'm confused how this would work? If we set `min_pin` to `x.x`, then wouldn't our pins effectively become `cudatoolkit>=11.5,<12`? How would this work for users that use `cudatoolkit=11.{0,2,4}`?

This is all assuming that we're only going to build packages with CUDA `11.5`, which is what I thought the plan was, no?